sdk/trace/batch_span_processor.go

```diff
@@ -265,7 +266,14 @@ func (bsp *batchSpanProcessor) exportSpans(ctx context.Context) error {
 	}
 
 	if l := len(bsp.batch); l > 0 {
-		global.Debug("exporting spans", "count", len(bsp.batch), "total_dropped", atomic.LoadUint32(&bsp.dropped))
+		global.Debug("exporting spans", "count", len(bsp.batch))
+
+		dropped := atomic.LoadUint32(&bsp.dropped)
+		knownDropped := atomic.SwapUint32(&bsp.knownDropped, dropped)
+		if knownDropped < dropped {
+			global.Warn("dropped spans", "total_dropped", dropped)
```

Comment:
```suggestion
			global.Warn("dropped spans", "total", dropped, "dropped", dropped-knownDropped)
```

Since we can now know how many spans were dropped since the last time spans were dropped, we might as well include that information in the log record.